include/WCSimOpRaman.hh

```diff
@@ -0,0 +1,188 @@
+// Modified copy of G4OpRayleigh.cc of geant4.10 to implement Raman scattering
+// Reference: APPLIED OPTICS / Vol. 37, No. 15 / 20 May 1998
+//
+// ********************************************************************
+// * License and Disclaimer                                           *
+// *                                                                  *
+// * The  Geant4 software  is  copyright of the Copyright Holders  of *
+// * the Geant4 Collaboration.  It is provided  under  the terms  and *
+// * conditions of the Geant4 Software License,  included in the file *
+// * LICENSE and available at  http://cern.ch/geant4/license .  These *
+// * include a list of copyright holders.                             *
+// *                                                                  *
+// * Neither the authors of this software system, nor their employing *
+// * institutes,nor the agencies providing financial support for this *
+// * work  make  any representation or  warranty, express or implied, *
+// * regarding  this  software system or assume any liability for its *
+// * use.  Please see the license in the file  LICENSE  and URL above *
+// * for the full disclaimer and the limitation of liability.         *
+// *                                                                  *
+// * This  code  implementation is the result of  the  scientific and *
+// * technical work of the GEANT4 collaboration.                      *
+// * By using,  copying,  modifying or  distributing the software (or *
+// * any work based  on the software)  you  agree  to acknowledge its *
+// * use  in  resulting  scientific  publications,  and indicate your *
+// * acceptance of all terms of the Geant4 Software license.          *
+// ********************************************************************
+//
+//
+// $Id: G4OpRayleigh.hh 84596 2014-10-17 07:34:41Z gcosmo $
+//
+// 
+////////////////////////////////////////////////////////////////////////
+// Optical Photon Rayleigh Scattering Class Definition
+////////////////////////////////////////////////////////////////////////
+//
+// File:        G4OpRayleigh.hh
+// Description: Discrete Process -- Rayleigh scattering of optical photons 
+// Version:     1.0
+// Created:     1996-05-31
+// Author:      Juliet Armstrong
+// Updated:     2014-08-20 allow for more material types
+//              2005-07-28 add G4ProcessType to constructor
+//              1999-10-29 add method and class descriptors
+//              1997-04-09 by Peter Gumplinger
+//              > new physics/tracking scheme
+// mail:        gum@triumf.ca
+//
+////////////////////////////////////////////////////////////////////////
```

Comment:
From what I can tell, you just copied the Rayleigh class, and renamed the class / rewrote the source
This was instead of setting up a brand new class that inherits from `G4VDiscreteProcess` directly
Both would give the same results (one is a bit quicker!) so I don't think this header is required
```suggestion
```